assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/index.test.js

```diff
@@ -230,6 +238,141 @@ describe( 'AudienceSelectionPanel', () => {
 				} );
 		} );
 
+		it.each( [
```

Comment:
While working on the issue, I realised that providing an additional story as per the IB wouldn't really add much value, as there would be no visual difference to the story.

I therefore took the approach of adding test coverage for the changes. This test is based on the one above it.

Reply:
Makes sense 👍